app/javascript/components/App.js

```diff
@@ -176,19 +176,19 @@ class App extends React.Component {
                   const singleEvent = this.state.parties.find(
                     (party) => party.id === id
                   );
-                  const partyItems = this.state.items.filter(
-                    (items) => items.party_id === id
-                  );
+
                   return (
                     <EventShow
                       party={singleEvent}
-                      items={partyItems}
                       id={id}
```

Comment:
You can also remove this prop because it's not being used and the party instance you care about is already provided to you through the party prop